plenum/server/node.py

```diff
@@ -2407,58 +2507,60 @@ def commitAndSendReplies(self, reqHandler, ppTime, reqs: List[Request],
             ppTime)
         return committedTxns
 
+    def default_executer(self, ledger_id, pp_time, reqs: List[Request],
+                         state_root, txn_root):
+        return self.commitAndSendReplies(
+            self.get_req_handler(ledger_id), pp_time, reqs, state_root,
+            txn_root)
+
     def executeDomainTxns(self, ppTime, reqs: List[Request], stateRoot,
```

Comment:
Can we use generic code for `executeXTxns`?
BTW looks like `executeDomainTxns` is overridden in indy-node, so the code about Roles is never executed?

Reply:
Yes, that is how it was before too, plenum keeps an in-memory map, indy-node does not

Reply:
Ok, so do you agree that we can have generic method here? There is a good work in this PR to use request handlers abstraction for doing similar works, so it looks like it may be also one of such cases.
But I'm OK if we do a change with `executeXTxns` in one of the next PRs.